assets/src/apps/delivery/layouts/deck/components/OptionsPanel.tsx

```diff
@@ -27,6 +27,9 @@ const OptionsPanel: React.FC<OptionsPanelProps> = ({ open }) => {
 
   return (
     <div className="optionsPanel">
+      <style type="text/css">
+        {'.displayOptionsView .option{margin-bottom: 10px; margin-top: 10px;}'};
```

Comment:
Each lesson has it's own external CSS file and we cannot go in each file and make these changes hence I am making the changes here. Again, this can be easily overridden from the external CSS if needed.